data/templates/centos-ks

```diff
@@ -53,6 +53,13 @@ sudo
 perl
 libselinux-python
 net-tools
+<% if( undefined !== kvm ) { %>
```

Comment:
I mean it's better to delete the "kvm" option if it is null, this is just to make the kickstart happy.

Meanwhile, we can also set the default value to "undefined", not "null".